content/applications/general/users/2fa.rst

```diff
@@ -1,127 +1,123 @@
+- `Microsoft Authenticator
+  <https://www.microsoft.com/en-gb/account/authenticator?cmp=h66ftb_42hbak>`_
 
-Phone-based authenticators are the easiest and most common so we will
-assume you'll pick and install one on your phone, examples include
-`Authy <https://authy.com/>`_, `FreeOTP
-<https://freeotp.github.io/>`_, `Google Authenticator
-<https://support.google.com/accounts/answer/1066447?hl=en>`_,
-`LastPass Authenticator <https://lastpass.com/auth/>`_, `Microsoft
-Authenticator
-<https://www.microsoft.com/en-gb/account/authenticator?cmp=h66ftb_42hbak>`_,
-...; password managers also commonly include :abbr:`2FA (two-factor
-authentication)` support e.g. `1Password
-<https://support.1password.com/one-time-passwords/>`_, `Bitwarden
-<https://bitwarden.com/help/article/authenticator-keys/>`_, ...
+Password managers are another option. Common examples include:
+- `1Password <https://support.1password.com/one-time-passwords/>`_
+- `Bitwarden <https://bitwarden.com/help/article/authenticator-keys/>`_,
 
-For the sake of demonstration we will be using Google Authenticator
-(not because it is any good but because it is quite common).
+.. note::
+   The remainder of this document uses Google Authenticator as an example, as it is one of the most
+   commonly used. This is not an endorsement of the product.
 
 Setting up two-factor authentication
 ====================================
 
-Once you have your authenticator of choice, go to the Odoo instance
-you want to setup :abbr:`2FA (two-factor authentication)`, then open
-:guilabel:`Preferences` (or :guilabel:`My Profile`):
+After selecting an authenticator, log in to Odoo, then navigate to :menuselection:`My Profile`.
 
-.. figure:: 2fa/preferences.png
-    :align: center
+Click the :guilabel:`Account Security` tab, then slide the :guilabel:`Two-Factor Authentication`
+toggle to *active*.
 
-Open the :guilabel:`Account Security` tab, then click the
-:guilabel:`Enable two-factor authentication` button:
-
-.. figure:: 2fa/sec_tab.png
+.. figure:: 2fa/account-security.png
    :align: center
 
-Because this is a security-sensitive action, you will need to input
-your password:
-
-.. figure:: 2fa/sec_enhanced.png
-   :align: center
+This generates a :guilabel:`Security Control` pop-up window that requires password confirmation to
+continue. Then, a :guilabel:`Two-Factor Authentication Activation` pop-up window appears, with a
+|QR|.
 
-After which you will see this screen with a barcode:
 
-.. figure:: 2fa/totp_scan.png
+.. figure:: 2fa/qr-code.png
    :align: center
 
-In most applications, you can simply *scan the barcode* via the
-authenticator of your choice, the authenticator will then take care of
-all the setup:
-
-.. figure:: 2fa/scan_barcode.jpg
-   :align: center
+Using the desired authenticator application, scan the |QR| when prompted.
 
 .. note::
+   If scanning the screen is not possible (e.g., because the set-up is being completed on the same
```

Comment:
```suggestion
   If scanning the screen is not possible (e.g. the setup is being completed on the *same*
```
here's my reasoning behind this suggestion:

'Set up' (two words) is a verb phrase, meaning it describes an action. For example, 'I will set up the computer.' 

'Setup' (one word) is a noun or an adjective, referring to a particular arrangement or configuration. For example, you might say "the setup of the computer is easy" or "this PC's setup is rather odd"